core/tee/tee_rpmb_fs.c

```diff
@@ -3081,5 +3081,5 @@ TEE_Result tee_rpmb_fs_raw_open(const char *fname, bool create,
 
 bool __weak plat_rpmb_key_is_ready(void)
 {
-	return true;
+	return false;
```

Comment:
If `plat_rpmb_key_is_ready()` is changed to always return `false`, then must all platforms that wishes to use RPMB override that function. If so then we could just as well remove it and require the platform to provide it if `CFG_RPMB_WRITE_KEY=y`. That will for sure trigger a lot of questions.

Overriding `plat_rpmb_key_is_ready()` is in my opinion only useful if there's some hardware state that can be checked to tell if it seems that the generated key will be the final key.

Reply:
Yes if the default is set to false, it will impact current users. But for some it might be for the best, protecting their RPMB hardware. For others is an inconvenience with no monetary cost...which could be worked around with an additional configuration.

When you work enabling hardware on different platforms, the current situation is nasty. Securing a board is not a trivial activity and this just adds pain to injury.

Reply:
It's a good point, but if we're to impact all users of RPMB we should try to make the best of it.
Is this the best thing we can do?
Do you have any comments to https://github.com/OP-TEE/optee_os/issues/5338?